web/libs/editor/src/tags/object/TimeSeries.jsx

```diff
@@ -1068,6 +1109,17 @@ const Overview = observer(({ item, data, series }) => {
     // give a bit more space for brush moving
     gb.current.select(".handle--w").style("transform", "translate(-1px, 0)");
     gb.current.select(".handle--e").style("transform", "translate(1px, 0)");
+
+    // Playhead cursor line
+    cursorLine.current = focus.current
+      .append("line")
+      .attr("class", "overview-playhead")
+      .attr("y1", 0)
+      .attr("y2", focusHeight)
+      .attr("stroke", item.cursorcolor || "#1e90ff")
```

Comment:
```suggestion
      .attr("stroke", item.cursorcolor || "var(--color-neutral-inverted-surface)")
```
@ricardoantoniocm what is the semantic default color we would use here for a playhead?

Reply:
We should use `--color-neutral-inverted-surface` in this instance.

This ensures consistency with the playhead of the audio player.
![image](https://github.com/user-attachments/assets/73a37662-315a-45e3-b53d-5d033c20dce3)

Red usually carries a negative/destructive connotation so we should avoid it unless there's a very good reason for it.